koku/masu/util/azure/azure_post_processor.py

```diff
@@ -59,7 +59,6 @@ class AzurePostProcessor:
         "ConsumedService",
         "ResourceType",
         "InstanceId",
-        "Tags",
         "OfferId",
         "AdditionalInfo",
         "ServiceInfo1",
```

Comment:
Do we need this or ServiceInfo2? Are we querying these columns for something in HCS?

Reply:
Yes we need this one.